internal/resolve/gosource/gosource_test.go

```diff
@@ -50,6 +60,14 @@ func TestResolve(t *testing.T) {
 			cwd, err := os.Getwd()
 			require.NoError(t, err)
 
+			if os.Getenv("GOCACHE") == "" {
+				os.Setenv("GOCACHE", t.TempDir())
```

Comment:
Why is GOCACHE set in the testcase?

Reply:
Without specifying a GOCACHE value this error occurs `exec: running 'C:\Go\bin\go.exe env -json' in directory '.' exited with code 1, expected 0, output: 'build cache is required, but could not be located: GOCACHE is not defined and %LocalAppData% is not defined'`

Reply:
Sounds to me like the root is that %LocalAppData% is not set in our execution environment and if we set it Go will be able to use the default GOCACHE directory.

Reply:
We probably have to whitelist `LocalAppData` here: 
https://github.com/simplesurance/baur/blob/d5dfe9cae22537dc6bd8ed592e5e996725ee6ac7/internal/resolve/gosource/gosource.go#L132

I'm  currently testing the change in a separate PR and push the commit to this branch if it works.

Reply:
works :-)
https://github.com/simplesurance/baur/pull/241/commits/c6d2f22d683dd3c2871795c38a3270366db0303b